content/05.dl-complexities.md

```diff
@@ -1,18 +1,20 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
 
-Correctly training deep neural networks is a non-trivial process.
-There are many different options and potential pitfalls at every stage.
-To get good results, you must expect to train many networks with a range of different parameter and hyperparameter settings.
-Deep learning can be very demanding, often requiring extensive computing infrastructure and patience to achieve state-of-the-art performance [@doi:10.1109/JPROC.2017.2761740].
-The experimentation inherent to DL is often noisy (requiring repetition) and represents a significant organizational challenge.
-All code, random seeds, parameters, and results must be carefully corralled using general good coding practices (for example, version control [@doi:10.1371/journal.pcbi.1004947], continuous integration etc.) in order to be effective and interpretable.
-This organization is also key to being able to efficiently share and reproduce your work [@doi:10.1371/journal.pcbi.1003285; @arxiv:1810.08055] as well as to update your model as new data becomes available.
+Correctly training deep neural networks is a non-trivial process, as there are many different options and potential pitfalls at every stage.
+To get good results, you must expect to train many networks across a range of different parameter and hyperparameter settings.
+Such training can be made more difficult by the demanding nature of these deep networks, which often require extensive computing infrastructure and optimization in order to achieve state-of-the-art performance [@doi:10.1109/JPROC.2017.2761740].
+Furthermore, this experimentation is often noisy, which necessitates increased repitition, and furthers the organizational challenges inherent to DL.
```

Comment:
Good catch, further is repeated. I tried to avoid increases since "increased" is 3 words before. How about "exacerbates", as below?

```suggestion
Furthermore, this experimentation is often noisy, which necessitates increased repitition, and exacerbates the organizational challenges inherent to DL.
```